.pre-commit-config.yaml

```diff
@@ -2,6 +2,12 @@
 # Please run `pre-commit run --all-files` when adding or changing entries.
 
 repos:
+  - repo: https://github.com/asottile/pyupgrade
+    rev: v3.15.0
+    hooks:
+      - id: pyupgrade
+        language_version: "3.9"
```

Comment:
Sometimes I just run this hook and then don't actually include it in the pre-commit-config. Just  sometime to consider in case the lint issues are hard to fix.

Reply:
Yeah, I'm a little wary of it getting in folks' way, but I'd also like to avoid another "do it sometimes" step a la `pytest --record-mode rewrite`. We can remove it later if we find that folks are running into problems.

Reply:
oh yeah I would have it be a "just do it when you bump up the min python version"